Source/Charts/Charts/BarChartView.swift

```diff
@@ -168,16 +170,16 @@ open class BarChartView: BarLineChartViewBase, BarChartDataProvider
     /// If enabled, highlighting operations will highlight the whole bar, even if only a single stack entry was tapped.
     @objc open var highlightFullBarEnabled: Bool = false
     
-    /// - returns: `true` the highlight is be full-bar oriented, `false` ifsingle-value
+    /// `true` the highlight is be full-bar oriented, `false` ifsingle-value
```

Comment:
why property does not need `return`?

Reply:
> why property does not need `return`?

![screen shot 2018-09-27 at 09 42 02](https://user-images.githubusercontent.com/11192474/46118563-c2af6880-c239-11e8-8f46-d993efab643b.png)
![screen shot 2018-09-27 at 09 42 43](https://user-images.githubusercontent.com/11192474/46118564-c347ff00-c239-11e8-85a5-aac9bc2f5b10.png)
![screen shot 2018-09-27 at 09 42 52](https://user-images.githubusercontent.com/11192474/46118565-c347ff00-c239-11e8-8ee5-3e0bba2f4c5c.png)

Cause property don't have `Returns` section. (and could not display it properly like function)